openff/toolkit/utils/openeye_wrapper.py

```diff
@@ -203,7 +203,7 @@ def is_available(cls) -> bool:
                     cls._is_installed = True
                     try:
                         importlib.import_module("openeye." + tool)
-                    except (ImportError, ModuleNotFoundError):
+                    except ImportError:
```

Comment:
I can't manage to get `importlib.import_module` to raise an `ImportError` - All of the following give me `ModuleNotFoundError`:

* `module = importlib.import_module("openeye.oechem")` raises no error, even when missing an OE license
* `module = importlib.import_module("openeye.zzz")` gives a `ModuleNotFoundError`
* `module = importlib.import_module("zzzz")` gives a `ModuleNotFoundError`

But I'm also not confident that this won't raise an `ImportError` - There's a lot of voodoo around python imports and OE Toolkit performance that I don't have a lot of confidence about. 

I'd propose that we revert these ones to `except (ImportError, ModuleNotFoundError):` and live with the bugbear we know.

Reply:
`ModuleNotFoundError` is a subclass of `ImportError`

```python3
>>> issubclass(ModuleNotFoundError, ImportError)
True
```

Installing OpenEye Toolkits doesn't require an active license, nor does importing `openeye.oechem`; if you uninstall it that will raise a `ModuleNotFoundError`

Reply:
Ahhh, thanks for the explanation!